pipeline_dp/aggregate_params.py

```diff
@@ -587,6 +586,35 @@ class PrivacyIdCountParams:
     contribution_bounds_already_enforced: bool = False
 
 
+@dataclass
+class AddDPNoiseParams:
+    """Specifies parameters for function DPEngine.add_dp_noise()
+
+   Attributes:
+       noise_kind: The type of noise to use for the DP calculations.
+       l0_sensitivity: the maximum number of partition for which 1 privacy unit
```

Comment:
Can we add something like: "Important: unlike the other methods, this method does not enforce the sensitivity by contribution bounding and relies on the caller to ensure the provided data satisfies the provided bound".

Also for the parameter below.